### PR TITLE
let the right and left list sync again

### PR DESCRIPTION
**Pull request by yingshaoxo**
This solution works for all kinds of `Headers`

Resolves: #49

### CODE REVIEW

File: lib/scroll/sync_scroll_controller_manager.dart [hunk 1]
```diff
@@ -81,7 +83,19 @@ class SyncScrollControllerManager {
   void _syncLeftListViewScrollVontroller(
       ScrollController _scrollingController, ScrollController controller) {
     if (_refreshController?.headerStatus != null) {
+      //log(_refreshController!.headerStatus.toString());
+
       switch (_refreshController?.headerStatus) {
+        case RefreshStatus.canRefresh:
+          {
+            //log(_scrollingController.offset.toString() + ", " + _refreshIndicatorHeight.toString());
```
Comment (MayLau-CbL) [inline lib/scroll/sync_scroll_controller_manager.dart:91]: Thanks @yingshaoxo ! I have tested web, android and ios which are all good. would you mind remove the log comment and the `import 'dart:developer';` line. then I will accept the PR and later release a new version.😉

Comment (yingshaoxo) [reply lib/scroll/sync_scroll_controller_manager.dart]: Hi, MyaLau-CbL. I just did what you asked. It's my pleasure to make a contribution.

**Review by MayLau-CbL — APPROVED** [review]

**Comment by MayLau-CbL** [comment]
Thank you again for the contribution. I will make a release for this tonight and will update you the status.

**Comment by MayLau-CbL** [comment]
This PR is available on the v3.4.1 on pub.dev.

**Pull request merged**